LDC cannot build Dfmt on Fedora Rawhide

I initially started an issue against Dfmt (https://github.com/Hackerpilot/dfmt/issues/129) as I could not get it to build using LDC (works fine with DMD). However, it may be that the problem is actually with my build of LDC as Brian can get a build of Dfmt using his build of LDC. From what I can see the difference is GCC 5.0 on Fedora Rawhide vs GCC 4.9.2 on Fedora.

I just did a complete, from scratch including cmake stage, build of LDC merge-2.067 (a22b2bbce0c6c3d70e5ed699c0e45b77997b6d31) which completes fine. When I try to compile Dfmt though I am getting the following link error. DCD compiles fine with this LDC compiler.

/home/russel/Built/lib/libphobos2-ldc-debug.a(curl-debug.o): In function `_D3std3net4curl4HTTP19_sharedStaticCtor16FZv':
/home/Checkouts/Git/Git/LDC/runtime/phobos/std/net/curl.d:2555: undefined reference to `curl_version_info'
/home/russel/Built/lib/libphobos2-ldc-debug.a(curl-debug.o): In function `_D3std3net4curl4Curl19_sharedStaticCtor18FZv':
/home/Checkouts/Git/Git/LDC/runtime/phobos/std/net/curl.d:3583: undefined reference to `curl_global_init'
/home/russel/Built/lib/libphobos2-ldc-debug.a(curl-debug.o): In function `_D3std3net4curl4Curl19_sharedStaticDtor19FZv':
/home/Checkouts/Git/Git/LDC/runtime/phobos/std/net/curl.d:3589: undefined reference to `curl_global_cleanup'
collect2: error: ld returned 1 exit status
Error: /usr/bin/gcc failed with status: 1

--- a/ldc2_install.conf.in
+++ b/ldc2_install.conf.in
@@ -10,7 +10,7 @@ default:
         "-I@INCLUDE_INSTALL_DIR@/ldc",
         "-I@INCLUDE_INSTALL_DIR@",
         "-L-L@CMAKE_INSTALL_LIBDIR@", @MULTILIB_ADDITIONAL_INSTALL_PATH@
-        "-defaultlib=phobos2-ldc,druntime-ldc",
-        "-debuglib=phobos2-ldc-debug,druntime-ldc-debug"
+        "-defaultlib=phobos2-ldc,curl,druntime-ldc",
+        "-debuglib=phobos2-ldc-debug,curl,druntime-ldc-debug"
     ];
 };

--- a/ldc2_phobos.conf.in
+++ b/ldc2_phobos.conf.in
@@ -14,7 +14,7 @@ default:
         "-L-L@PROJECT_BINARY_DIR@/../lib@LIB_SUFFIX@/Debug",
         "-L-L@PROJECT_BINARY_DIR@/../lib@LIB_SUFFIX@/Release",
         "-L-L@PROJECT_BINARY_DIR@/../lib@LIB_SUFFIX@/RelWithDebInfo",
-        "-defaultlib=phobos2-ldc,druntime-ldc",
-        "-debuglib=phobos2-ldc-debug,druntime-ldc-debug"
+        "-defaultlib=phobos2-ldc,curl,druntime-ldc",
+        "-debuglib=phobos2-ldc-debug,curl,druntime-ldc-debug"
     ];
 };

